user-interface/src/data-verification/consolidation/ConsolidationOrderAccordion.test.tsx

```diff
@@ -267,10 +230,11 @@ describe('ConsolidationOrderAccordion tests', () => {
     });
   });
 
-  test('should correctly enable/disable buttons when selecting consolidated cases and lead case from order case list table', async () => {
+  test.skip('should correctly enable/disable buttons when selecting consolidated cases and lead case from order case list table', async () => {
```

Comment:
We're skipping a lot of tests in this file.  We should determine if we can throw them away or if they need to be fixed.